src/SprykerEco/Zed/UnzerApi/Business/Api/Request/Converter/UpdateCustomerRequestConverter.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerApi\Business\Api\Request\Converter;
+
+use Generated\Shared\Transfer\UnzerAddressTransfer;
+use Generated\Shared\Transfer\UnzerApiRequestTransfer;
+use SprykerEco\Zed\UnzerApi\Business\Api\Request\UnzerApiRequestConstants;
+
+class UpdateCustomerRequestConverter implements UnzerApiRequestConverterInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\UnzerApiRequestTransfer $unzerApiRequestTransfer
+     *
+     * @return array<string, array<string, string|null>|string
```

Comment:
Looks incorrect, because any `get` method in transfer can return `null`. And don't forget `>` at the end.